.github/workflows/clean_validation_envs.yml

```diff
@@ -29,6 +29,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           MAIN_TRE_ID: ${{ secrets.TRE_ID }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
```

Comment:
In other places we just use the github context variables in scripts without the need to pass them explicitly in the env. Have you check this here and it didn't work?

Reply:
I tried at first without at first and it didn't work, and then I saw we pass it this way in all other places. Are you familiar with an example?